File: crackme.java
import java.util.Scanner;

public class main{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("password: ");
        String password = input.nextLine();
        if (password.equals("crackme")) {
            System.out.println("success!");
        } else {
            System.out.println("fail.");
        }
    }
}

File: crackme.rs
use std::io;
use std::io::Write;

fn main() {
    let mut input = String::new();

    print!("password: ");
    io::stdout().flush().unwrap();

    io::stdin().read_line(&mut input)
        .expect("Failed to read line");

    if input.trim() == "crackme" {
        println!("success!");
    } else {
        println!("fail.");
    }
}